test/integration/bridges/LZRsEthBridge.t.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: BSL-1.1
+pragma solidity =0.8.28;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {LZRsEthBridge} from "src/rebalancer/bridges/deprecated/LZRsEthBridge.sol";
+import {Base_Integration_Test} from "../Base_Integration_Test.t.sol";
+import {MockRoles} from "test/mocks/MockRoles.sol";
+
+import {LZOptions} from "src/libraries/LZOptions.sol";
+import "forge-std/console.sol";
+
+contract MockMarket {
+    address public underlying;
+    constructor(address _underlying) {
+        underlying = _underlying;
+    }
+}
+
+contract LZUnifiedBridgeIntegrationTest is Base_Integration_Test {
```

Comment:
Add integration that exercises a wrapper/adapter path with compose enabled and asserts unwrap + delivery to the market, and asserts refunds reach the payer.